cmake/modules/AddSwift.cmake

```diff
@@ -618,8 +618,7 @@ function(_add_swift_library_single target name)
   else()
     string(REPLACE swift "" module_name "${name}")
   endif()
-  list(FIND SWIFT_API_NOTES_INPUTS "${module_name}" overlay_index)
-  if(NOT ${overlay_index} EQUAL -1)
+  if("${modue_name}" IN_LIST SWIFT_API_NOTES_INPUTS)
```

Comment:
And sure enough, a completely clean build did reproduce this on my macOS machine. Geez! 😬